CDS Maps/Individual Maps/FHIR-to-CDS/nNGM-FHIR-to-CDS-Strahlentherapie.map

```diff
@@ -1,431 +1,282 @@
 /// version = 0.1
 /// title = "nNGM: Sonstige Therapie - FHIR to CTS"
 
+// TODO: find out why output is produced twice 
```

Comment:
Is this still relevant? If so, we should solve it first 👍

Reply:
not relevant anymore